### PR TITLE
Added the command to update Doctrine schema

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Hello,

for any Symfony developer, this task is obvious but for beginners it's not :)

I think it's better to display the command to execute after the completion of the override.

Mickaël

### CODE REVIEW

File: cookbook/catalog_structure/overriding_the_orm_product_value.rst [hunk 1]
```diff
@@ -269,4 +269,8 @@ Finally, check that your mapping override is correct by launching the following
 
     php app/console doctrine:mapping:info
 
-Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class.
+Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class:
+
+.. code-block:: bash
+
+    php app/console doctrine:schema:update --force
```
Comment (rybus) [inline cookbook/catalog_structure/overriding_the_orm_product_value.rst]: I think appending `--dump-sql` on this command would be a plus just to know what has been done.

Comment (mickaelandrieu) [inline cookbook/catalog_structure/overriding_the_orm_product_value.rst]: Agree, update done!

File: cookbook/catalog_structure/overriding_the_orm_product_value.rst [hunk 2]
```diff
@@ -269,4 +269,8 @@ Finally, check that your mapping override is correct by launching the following
 
     php app/console doctrine:mapping:info
 
-Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class.
+Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class:
+
+.. code-block:: bash
+
+    php app/console doctrine:schema:update --dump-sql --force
```
Comment (fitn) [inline cookbook/catalog_structure/overriding_the_orm_product_value.rst]: I think commands should be always written in prod environment but I can see upper that it is not the case. What are you thinking about that?
It also needs a cache clear and apache2 restart on production environment

File: cookbook/catalog_structure/overriding_the_orm_product_value.rst [hunk 3]
```diff
@@ -269,4 +269,8 @@ Finally, check that your mapping override is correct by launching the following
 
     php app/console doctrine:mapping:info
 
-Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class.
+Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class:
```
Comment (MarieMinasyan) [inline cookbook/catalog_structure/overriding_the_orm_product_value.rst]: to use your own

File: cookbook/catalog_structure/overriding_the_orm_product_value.rst (hunk 2 above)
Comment (rybus) [inline cookbook/catalog_structure/overriding_the_orm_product_value.rst]: I don't think that in that case it is important ;)

Comment (mickaelandrieu) [inline cookbook/catalog_structure/overriding_the_orm_product_value.rst]: I assume the developer is in development mode and is aware of environments and cache issues: if it's not the case, this needs a specific chapter somewhere... but not here. What do you think?

File: cookbook/catalog_structure/overriding_the_orm_product_value.rst (hunk 3 above)
Comment (mickaelandrieu) [inline cookbook/catalog_structure/overriding_the_orm_product_value.rst]: true! I update

**Comment by mickaelandrieu** [comment]
Thanks for review all!

**Pull request merged**